clip.js

```diff
@@ -75,13 +79,15 @@
 
         // Workaround to trigger an event when the image set as background
         // via CSS finishes loading
-        const img = new Image();
-        // We need a reference to the Collage instance to be able to
-        // instead call self.getLibraryFilePath('h5p.svg'). This is likely
-        // good enough.
-        img.src = H5P.getLibraryPath('H5P.Collage-0.3') + '/h5p.svg';
-        img.onload = () => {
-          self.trigger('loaded');
+        if (
+          typeof self.parent?.getLibraryFilePath === 'function' &&
+          self.parent?.libraryInfo
```

Comment:
@Languafe : In which context isn't .getLibraryFilePath and .libraryInfo available?